packages/contracts/README.md

```diff
@@ -160,5 +160,23 @@ If you are using a network which Etherscan supports you can verify your contract
 npx hardhat etherscan-verify --api-key ... --network ...
 ```
 
+### Other hardhat tasks
+
+To whitelist deployers on Mainnet you must have the whitelist Owner wallet connected, then run:
+```bash
+npx hardhat whitelist \
+  --use-ledger true \  # `network` MUST equal your env var `CONTRACTS_TARGET_NETWORK`
+  --contracts-rpc-url https://mainnet.optimism.i \
+  --address ... \ # address to whitelist
+```
+
+To withdraw ETH fees to L1 on Mainnet, run:
+```bash
+npx hardhat withdraw-fees \
+  --private-key 0x... \  # The private key to withdraw fees with. Ensure this wallet has ETH on L2 to pay the tx fee.
```

Comment:
nit: move comments out of the code snippet and above so that it is easier to copy/paste the command